lca-cli/cmd/staterootSetupIBU.go

```diff
@@ -0,0 +1,104 @@
+package cmd
```

Comment:
nit: suggest using IBU as a prefix, rather than suffix... ie. ibuStaterootSetup.go, ibuStaterootSetupCmd, etc

Reply:
Updated to `ibuStaterootSetup.go`